### ISSUE TITLE
Unable to copy from sqlite browser.

### ISSUE THREAD

**Issue by aurangzaibumer777** [sqlitebrowser/sqlitebrowser#2898]
![Screenshot 2021-11-08 20:48:55](https://user-images.githubusercontent.com/33724757/140773641-b4ac7dcd-de4c-49df-b7ea-c87f84047dfc.png)

**Comment by chrisjlocke** [comment]
What details are you trying to cut/copy?  From the 'browse data' grid?  Or from the 'Execute SQL' grid?
Either way, results from the grid should be copyable (if that was a word...)
That doesn't look like a Windows screenshot, nor macOS ... so I assume you're on some variant of Linux?

**Comment by chrisjlocke** [comment]
It 'should' work for Linux in the same way, but this is how it works on Windows...

https://user-images.githubusercontent.com/7872218/140778936-301df436-0bd9-490b-a530-39565e83b181.mp4

**Comment by aurangzaibumer777** [comment]
Yes using Linux as mentioned. Yes tried everything from one column to the whole grid. Not working at all

**Comment by chrisjlocke** [comment]
We've not had other reports, so I'd be more inclined to blame your system than our software! ;)
I assume you've rebooted recently, or you've tried in a clean environment (eg, after a new reboot) just in case it's other software that's interfering somehow...
I'm not a Linux user, but have you got a lot running in the background?  I know some anti-virus software can be 'aggressive' by monitoring the clipboard data and clearing it for 'privacy' reasons, etc.
